src/implementations/vanilla/components/site-header/site-header-eu.scss

```diff
@@ -78,16 +78,12 @@ $_search-width-xl: 31.5rem;
   flex-grow: 1;
 }
 
-.ecl-site-header__logo-image-mobile {
```

Comment:
if we remove the previous css class definition, that would be a breaking change
So we should either keep them (deprecated) or flag the pr as breaking